src/TimeKeeping.Core/Data/Entities/ClockRoll.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Th11s.TimeKeeping.Data.Entities
+{
+    public class RollingClockEntry
+    {
+        public Guid Uuid { get; set; }
+
+        public int UserId { get; set; }
+        public int OrgUnitId { get; set; }
+
+
+        public required Timestamp Stamp { get; set; }
+        public AggregationInfo? Aggregation { get; set; }
```

Comment:
Bekommen die Aggregate jeweils eine eigene Tabelle? Wofür genau ist diese Info da? Wird die jedes mal gesetzt wenn ein neues Aggregat erstellt wird? Also auch wenn man von Tages => Wochen geht oder nur von Eintrag => Tag?

Reply:
Ich dachte an Tages, Monats- und Jahressummen, die dann jeweils auch die Dokumente entsprechend zusammenfassen.
Wochen müssen wir "live" berechnen, da sie nicht in den Jahreszyklus passen und es also kein Tag -> Woche -> Monat -> Jahr geben kann.

Soll ist also Tag -> Monat -> Jahr, jeweils mit Aggregatinfo. Damit kann man "Neuberechnungen" schlank halten, da man weiß, welche Elemente bereits im Aggregat enthalten sind und welche ein Update brauchen - Delta berechnen, statt vollabgleich sozusagen.

Reply:
Also sollen bei Änderungen der Einträge auch direkt die Aggregate geändert werden?

Reply:
Das ist ggf nicht möglich - die Einträge müssen erst konsistent sein, daher würde ich das in der Regel nicht machen.
Eventuell kann man die Aggregate als "dreckig" markieren....

Reply:
Du brauchst ja mindestens 2 Einträge für ein Tagesaggregat.

Reply:
Die Frage hat eher darauf abgezielt, dass wenn du bereits eine Aggregatsinfo hast und der Entry gelöscht wird, muss der ja trotzdem beim nächsten Aggregatupdate berücksichtigt werden. Genauso wenn einer eine Aggregatsinfo hat aber sich geändert hat. Bezogen auf das Berechnen des Delta